docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md

```diff
@@ -104,3 +104,10 @@ To create an export configuration, do the following:
 To view the configurations, click the **View** button for the provider.
 
 To delete a configuration, click **View** and choose **Delete**.
+
+If you need to replace or update an API key for an integration for a particular tool, do the following:
```

Comment:
There are other things customer may want to edit in their integrations(among the 4 integrations we support right now there are a lot of configurables), so can the message be something along the lines of
`If you need to edit your integration for any purpose such as to replace or update an API key..`